osu.Game/Skinning/ArgonSkin.cs

```diff
@@ -166,8 +168,20 @@ public ArgonSkin(SkinInfo skin, IStorageResourceProvider resources)
 
                                     if (songProgress != null)
                                     {
-                                        songProgress.Position = new Vector2(0, -10);
+                                        const float padding = 10;
+
+                                        songProgress.Position = new Vector2(0, -padding);
                                         songProgress.Scale = new Vector2(0.9f, 1);
+
+                                        if (keyCounter != null && hitError != null)
+                                        {
+                                            // Hard to find this at runtime, so taken from the most expanded state during replay.
+                                            const float song_progress_offset_height = 36 + padding;
+
+                                            keyCounter.Anchor = Anchor.BottomRight;
+                                            keyCounter.Origin = Anchor.BottomRight;
+                                            keyCounter.Position = new Vector2(-(hitError.Width + padding), -(padding * 2 + song_progress_offset_height));
```

Comment:
Does it means that we are discarding the Figma design? It was expected that the key counter was positioned to the left and another Argon variant of the judgment counter to the right according to the design. 

![IMG_20230626_094407.jpg](https://github.com/ppy/osu/assets/39100084/3293c9c7-3433-4259-b4b1-53569a0a6693)

![IMG_20230626_094648.jpg](https://github.com/ppy/osu/assets/39100084/f4aacd0b-44cb-402d-82a1-2417ae584c26)

What is better between stable expectations and the Figma design feels blurry to me tbh.